src/RoleObjects/AccessibilityObject.ts

```diff
@@ -52,6 +51,8 @@ export default class AccessibilityObject {
 
   private _visible: boolean;
 
+  private _markedForUpdate: boolean;
```

Comment:
Small nit: properties can have default values set when they're defined, rather than having to do it in the constructor:
```ts
private _markedForUpdate: boolean = false;
```